src/installing-workflow/configuring-object-storage.md

```diff
@@ -11,11 +11,13 @@ A variety of Deis components rely on an object storage system to do their work.
 
 These components are flexible and can work out of the box with almost any system that is compatible with the [S3 API](http://docs.aws.amazon.com/AmazonS3/latest/API/APIRest.html).
 
-Note: object storage configuration has not been standardized across all components in our beta release. As such, configuration instructions differ for each component. We plan to remediate this problem in our next release. Please see our [deis/deis#4966](https://github.com/deis/deis/issues/4966) for our progress on that work.
+Note: object storage configuration has not been standardized across all components in our beta release. As such, configuration instructions differ for each component. We plan to remediate this problem in our next release. Please see [deis/deis#4966](https://github.com/deis/deis/issues/4966) for our progress on that work.
 
 ## Minio
 
-Additionally, Deis ships with a [Minio](http://minio.io) [component](https://github.com/deis/minio). This component runs as a Kubernetes service, and the components listed above are configured to automatically look for that service and use it as object storage if it's available.
+Additionally, Deis ships with a [Minio](http://minio.io) [component](https://github.com/deis/minio) by default, which provides in-cluster, ephemeral object storage. This means that _if the Minio server crashes, all data will be lost forever_. Therefore, **Minio should be used for development or testing only**.
```

Comment:
Maybe take out the word "forever"? It's not wrong, but it sounds a bit too dramatic IMHO.